### PR TITLE
Multi reference

### PR DESCRIPTION
**Pull request by holtjma**
- Adds reference specification since this is subject to change with GIAB releases
- Reorganization of some files along with backing up some of the paper scripts while in here

### CODE REVIEW

File: scripts/paper_analysis/AnalyzeBedFile.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            fpo.write(l)
+        else:
+            empty += 1
+    
+    fpo.close()
+    fp.close()
+    print(found, empty, sep='\t')
+
+if __name__ == "__main__":
+    vcf1 = '/gpfs/gpfs1/home/jholt/sanger_less_tests/data/HG001_GRCh38_GIAB_highconf_CG-IllFB-IllGATKHC-Ion-10X-SOLID_CHROM1-X_v.3.3.2_highconf_PGandRTGphasetransfer.vcf.gz'
```
Comment (rjones-haib) [inline scripts/paper_analysis/AnalyzeBedFile.py:36]: gpfs reference

File: scripts/paper_analysis/AnalyzeBedFile.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    bed1 = '/gpfs/gpfs1/home/jholt/sanger_less_tests/data/HG001_GRCh38_GIAB_highconf_CG-IllFB-IllGATKHC-Ion-10X-SOLID_CHROM1-X_v.3.3.2_highconf_nosomaticdel_noCENorHET7.bed'
```
Comment (rjones-haib) [inline scripts/paper_analysis/AnalyzeBedFile.py:37]: gpfs reference

File: scripts/paper_analysis/AnalyzeBedFile.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    
+    filterBedDir = '/gpfs/gpfs1/home/jholt/sanger_less_tests/data/bed_empty_filtered'
```
Comment (rjones-haib) [inline scripts/paper_analysis/AnalyzeBedFile.py:39]: gpfs reference

File: scripts/paper_analysis/AnalyzeBedFile.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    if not os.path.exists(filterBedDir):
+        os.makedirs(filterBedDir)
+
+    for x in range(1, 8):
+        
+        #print(f'/gpfs/gpfs1/home/jholt/sanger_less_tests/data/HG00{x}*GRCh38*.vcf.gz')
+        candVCF = glob.glob(f'/gpfs/gpfs1/home/jholt/sanger_less_tests/data/HG00{x}*GRCh38*.vcf.gz')
```
Comment (rjones-haib) [inline scripts/paper_analysis/AnalyzeBedFile.py:46]: gpfs reference

File: scripts/paper_analysis/AnalyzeBedFile.py [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        if x == 1:
+            candVCF = [vcf1]
+        assert(len(candVCF) == 1)
+        candBED = glob.glob(f'/gpfs/gpfs1/home/jholt/sanger_less_tests/data/HG00{x}*GRCh38*.bed')
```
Comment (rjones-haib) [inline scripts/paper_analysis/AnalyzeBedFile.py:50]: gpfs reference

File: scripts/paper_analysis/AnalyzeFPDistance.py [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+    #aligner = 'dragen-07.011.352.3.2.8b'
+    #caller = 'dragen-07.011.352.3.2.8b'
+    
+    aligner = 'dragen-07.021.510.3.5.7'
+    caller = 'dragen-07.021.510.3.5.7'
+
+    #aligner = 'clinical_sentieon-201808.07'
+    #caller = 'strelka-2.9.10'
+    
+    print(f'/gpfs/gpfs1/home/jholt/csl_validations/core_pipeline_analysis/pipeline/rtg_results/{aligner}/{caller}/SL*/fp.vcf.gz')
```
Comment (rjones-haib) [inline scripts/paper_analysis/AnalyzeFPDistance.py:30]: gpfs reference

File: scripts/paper_analysis/AnalyzeFPDistance.py [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+    vcfList = sorted(glob.glob(f'/gpfs/gpfs1/home/jholt/csl_validations/core_pipeline_analysis/pipeline/rtg_results/{aligner}/{caller}/SL*/fp.vcf.gz'))
```
Comment (rjones-haib) [inline scripts/paper_analysis/AnalyzeFPDistance.py:31]: gpfs reference

File: scripts/paper_analysis/CostVariants.py [hunk 8]
```diff
@@ -0,0 +1,192 @@
...
+    variants = []
+    for variant in vcfReader:
+        var = ('chr'+variant.CHROM, str(variant.POS), str(variant.POS), variant.REF, ';'.join([str(s) for s in variant.ALT]))
+        fpo.write(','.join(var)+'\n')
+        variants.append(var)
+    fpo.close()
+    return variants
+
+def runCostAnalysis(slid, variantFN):
+    resultFN = '/gpfs/gpfs1/home/jholt/sanger_less_tests/results/cost_results_v2/%s.tsv' % slid
```
Comment (rjones-haib) [inline scripts/paper_analysis/CostVariants.py:62]: gpfs reference

File: scripts/paper_analysis/CostVariants.py [hunk 9]
```diff
@@ -0,0 +1,192 @@
...
+    if not os.path.exists(resultFN):
+        cmdFrags = [
+            'python3', '-u', 'EvaluateVariants.py',
+            '-c', variantFN,
+            '-m', 'clinical_v2',
+            '-r', '0.995',
+            '-o', resultFN,
+            '/gpfs/gpfs1/home/jholt/sanger_less_tests/clinical_models/dragen-clinical-v1.0.0',
```
Comment (rjones-haib) [inline scripts/paper_analysis/CostVariants.py:70]: gpfs reference

File: scripts/paper_analysis/CostVariants.py [hunk 10]
```diff
@@ -0,0 +1,192 @@
...
+            '/gpfs/gpfs2/dragen/clinical/v0/%s/%s.hard-filtered.gvcf.gz' % (slid, slid)
```
Comment (rjones-haib) [inline scripts/paper_analysis/CostVariants.py:71]: gpfs reference

File: scripts/paper_analysis/CostVariants.py [hunk 11]
```diff
@@ -0,0 +1,192 @@
...
+    #optional arguments with default
+    p.add_argument('-v', '--variants', dest='variants', default=None, help='variant coordinates to evaluate (default: None)')
+    #p.add_argument('-m', '--model', dest='model', default='best', help='the model name to use (default: best)')
+    #p.add_argument('-r', '--recall', dest='recall', default='0.995', help='the target recall value from training (default: 0.995)')
+    #p.add_argument('-o', '--output', dest='outFN', default=None, help='the place to send output to (default: stdout)')
+
+    #required main arguments
+    p.add_argument('model_directory', type=str, help='directory with models and model stats')
+    #p.add_argument('sample_vcf', type=str, help='VCF file with variants to evaulate')
+    PATTERN = '/gpfs/gpfs2/dragen/clinical/v0/{slid}/{slid}.hard-filtered.gvcf.gz'
```
Comment (rjones-haib) [inline scripts/paper_analysis/CostVariants.py:134]: gpfs reference

**Review by rjones-haib — CHANGES_REQUESTED** [review]
Several gpfs references. Otherwise looks good to me

**Comment by holtjma** [comment]
yea, the gpfs references are fine, we're just saving these for records; it's unlikely they'll be used again

**Pull request merged**